### DISCUSSION TITLE
Run quietly

### DISCUSSION THREAD

**Discussion by lambdawalker** [behave/behave#1033] [Q&A] [answered]
This might be a silly question.

I have tried to run from the terminal like this: `python -m behave ... `

I have used all the modifiers related to logging and didn't find a combination to avoid the features and scenarios printed on the terminal.

```

Feature: Assert Equals

  Scenario: As a dev I want to display custom messages when assertion fails so that I have better information on failing cases
    Given a custom message and to objects that are not equal
    When assert_equals is executed with the given parameters
    Then the exception message should start with the custom message

  Scenario: As a dev I want to format subjects under test when assertion fails so that I have better information on failing cases
    Given a custom format method and to objects that are not equal
    When assert_equals is executed with the given parameters
    Then the parameters in the exception message should be formatted accordingly

1 feature passed, 0 failed, 0 skipped
2 scenarios passed, 0 failed, 0 skipped
6 steps passed, 0 failed, 0 skipped, 0 undefined
Took 0m0.002s

```

I'm using behave 1.2.6.

What is the right combination of modifiers to avoid verbosity in the terminal?

**Comment by bittner** [comment, marked as answer]
I'm not sure about silencing completely, but have you tried `--format=progress` for less output? Maybe that already helps?

See the [Formatters and Reporters](https://behave.readthedocs.io/en/stable/formatters.html?highlight=progress) chapter of the documentation for background reading.
